{
  "contest": 95,
  "handle": "sorrynotsorry",
  "address": "0x6262f720679C04cc7f2a14c0c03486D0287c4853",
  "risk": "2",
  "title": "Re-entrancy  at FiatTokenV1.sol",
  "issueId": 37,
  "issueUrl": "https://github.com/code-423n4/2022-02-jpyc-findings/issues/37"
}